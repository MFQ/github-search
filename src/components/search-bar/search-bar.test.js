import React from 'react';
import { shallowToJson } from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';

import SearchBar from ".";

configure({ adapter: new Adapter() });

describe('SearchBar', () => {
  it('should render correctly', () => {
    const mockData = {
      bids: [],
      asks: []
    }
    const output = shallow(
      <SearchBar/>
    );
    expect(shallowToJson(output)).toMatchSnapshot();
  });
});
