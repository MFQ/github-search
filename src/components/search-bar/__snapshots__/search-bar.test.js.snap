// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBar should render correctly 1`] = `
<div>
  <h1>
     Search Github Repos 
  </h1>
  <InputGroup
    size="lg"
  >
    <FormControl
      aria-describedby="inputGroup-sizing-sm"
      aria-label="Large"
      onBlur={[Function]}
      onChange={[Function]}
      value=""
    />
  </InputGroup>
</div>
`;
